spec/lib/http/options/headers_spec.rb

```diff
@@ -14,7 +14,11 @@
   end
 
   it "accepts any object that respond to :to_hash" do
-    x = Struct.new(:to_hash).new("accept" => "json")
+    x = if RUBY_VERSION >= "3.2.0"
+          Data.define(:to_hash).new(:to_hash => { "accept" => "json" })
+        else
+          Struct.new(:to_hash).new({ "accept" => "json" })
+        end
```

Comment:
`Struct.new(:to_hash, keyword_init: false).new("accept" => "json")`

If this is the case, the code will not branch by version, but keyword_init: false will cause rubocop to say `Use hash rockets syntax.

Reply:
```suggestion
    Struct.new(:to_hash, keyword_init: false).new("accept" => "json")
```

`Struct.new(:to_hash, keyword_init: false).new("accept" => "json")`

If this is the case, the code will not branch by version, but keyword_init: false will cause rubocop to say `Use hash rockets syntax.